cmd/schedule-builder/cmd/markdown.go

```diff
@@ -60,6 +61,52 @@ func parseSchedule(patchSchedule PatchSchedule) string {
 	return scheduleOut
 }
 
+// runs with `--type-file=release` to retrun the release cycle schedule
+func parseReleaseSchedule(releaseSchedule ReleaseSchedule) string {
+	output := []string{}
+	output = append(output, fmt.Sprintf("# Kubernetes %s\n", releaseSchedule.Releases[0].Version))
```

Comment:
I think it would be good to have a base go template for the layout: https://pkg.go.dev/text/template

Reply:
Thanks a lot for this suggestion, I have tried to read blogs like (https://golangdocs.com/templates-in-golang) and official docs but I have to still try how it will work on our case, if you have any tips for that, it would be really great to set the task for it then. Thanks :blush:

Reply:
First of all we would need a template string which can be later on filled with the right values from functions or structures. Do you think you could provide a starting point like this?

Reply:
Sure thing, I will try to work out on this today and maybe send a gist initially just to check how our starting point looks, thank you so much!